.github/workflows/build-and-test.yml

```diff
@@ -1,6 +1,9 @@
 ﻿name: build and test
-on: [push, pull_request]
-
+on:
+  pull_request:
+  push:
+    tags-ignore:
```

Comment:
I got tripped up here at first cause the conditional is in two places. Couldn't we always "build-and-test" and then on certain occassions _push_-backend ?  Maybe I'm just not familiar with actions and seeing the same conditional statement in two places (tags-ignore: Backend & tags: Backend). Looks like we also ignore tests entirely on the backend tagged version.